common/changes/@microsoft/rush/sachinjoseph-lazyLoadLocalProjects_2020-08-19-18-36.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Load and validate local projects lazily to further improve Rush startup times.",
```

Comment:
@sachinjoseph Suppose that there is an error somewhere in the project files. Would this make it possible to for someone to merge a PR with broken config files, whereas prior to your PR it would have been caught?

Reply:
`rush change` and `rush change --verify` both access these lazy-loaded properties, so the local `rush change` and the PR gate will be able to catch any validation fails.

Reply:
actually, `rush install` would also need to access these lazy-loaded properties, so the PR build should be able to catch any validation fails.

Reply:
@sachinjoseph Cool.  

FYI `rush change` would NOT be an okay validation, because a developer might invoke it before introducing the error.  Whereas `rush install` is good, because it always runs in CI and can fail the CI build.